frontend/__tests__/components/NetworkVisualization.test.tsx

```
@@ -19,40 +19,564 @@ jest.mock('react-plotly.js', () => {
+        { id: "1", name: "N1", symbol: "S1", asset_class: "EQUITY" } as any,
+      ], // missing x,y,z
+      edges: [
+        {
+          source: "1",
+          target: "2",
+          relationship_type: "TEST",
+          strength: 0.5,
+        } as any,
+      ],
+    };
+    render(<NetworkVisualization data={dataWithMissingCoords} />);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)